pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -442,6 +449,16 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 		return true, requeueAfter, nil
 	}
 
+	// Check if revision should be installed based on if the quorum is about to be violated
+	if c.shouldRevisionInstall != nil {
```

Comment:
as far as I understand, this blocks the entire controller, which we don't want, we want the current revisions to continue to rollout. 

We can be more granular here and just block the `ensureInstallerPod` routine from spawning just the installer pod. That way it wouldn't block existing rollouts. Having said that, I think there are generally very little tests around this, so we need to run a couple of payload tests in cluster-etcd-operator (CEO) as well